### PR TITLE
Chessboard

### PR DESCRIPTION
**Pull request by twbeasley**
Added Chessboard to games how page
Added pieces to chessboard

### CODE REVIEW

**Review by lisarics — APPROVED** [review]
Nicely done.

File: app/models/rook.rb [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+  end
+   
+  
+  
+ def valid_move?(board, stop)
+  if self.is_vertical_move?(board,stop)
+    return true
+  elsif self.is_horizontal_move?(board, stop)
+    return true
+  end
```
Comment (jakekarl3) [inline app/models/rook.rb:11]: If we're going to use an if statement with an elsif, there should be an else statement as well. Currently in the case that the move is not vertical or horizontal, the loop will go to the third option (else) which isn't specified so it'll default to nil which we don't want sent back to whatever method is calling this one.
Alternatively, this method could be rewritten with the single line as:

```
def valid_move?(board, stop)
  self.is_vertical_move?(board, stop) || self.is_horizontal_move?(board, stop)
end

File: app/models/bishop.rb [hunk 2]
```diff
@@ -1,2 +1,10 @@
 class Bishop < ChessPiece
+  def display
+    if color == "black"
+    return "&#9821;"
+    else 
+      return "&#9815;"
+
```
Comment (jakekarl3) [inline app/models/bishop.rb]: We should clean up the extra white space and make sure the indentations are formatted correctly

File: app/models/king.rb [hunk 3]
```diff
@@ -1,2 +1,9 @@
 class King < ChessPiece
+  def display
+    if color == "black"
+      return "&#9818;"
+    else
+      return "&#9812;"
+  end
```
Comment (jakekarl3) [inline app/models/king.rb:7]: Make sure the indentations are formatted correctly

File: app/models/knight.rb [hunk 4]
```diff
@@ -1,2 +1,9 @@
 class Knight < ChessPiece
+  def display
+    if color == "black"
+      return "&#9822;"
+    else
+      return "&#9816;"
+  end
```
Comment (jakekarl3) [inline app/models/knight.rb:8]: Make sure the indentations are formatted correctly

File: app/models/pawn.rb [hunk 5]
```diff
@@ -1,2 +1,8 @@
 class Pawn < ChessPiece
+  def display
+    if color == "black"
+      return "&#9823;"
+    else "&#9817;"
+  end
```
Comment (jakekarl3) [inline app/models/pawn.rb:8]: Should follow the same structure with a return "&#9817" being under the else statement.

File: app/models/queen.rb [hunk 6]
```diff
@@ -1,2 +1,8 @@
 class Queen < ChessPiece
+  def display
+    if color == "black"
+      return "&#9819;"
+    else "&#9813"
```
Comment (jakekarl3) [inline app/models/queen.rb]: Should follow the same structure with a return "&#9813" being under the else statement.

**Pull request merged**